Sources/PackageModel/Product.swift

```diff
@@ -41,11 +41,6 @@ public class Product {
     }
 }
 
-public enum LibraryType {
-    case dynamic
-    case Static
-}
-
```

Comment:
You are free to land these two diffs on master directly if they are really just trivial refactors (or land in a separate PR you send through test-and-merge).